lib/datasource/dart/index.js

```diff
@@ -0,0 +1,57 @@
+const got = require('got');
```

Comment:
Please use `lib/util/got.js` - check other datasources for examples. That way per-run caching is automatically enabled.

Reply:
`got` it :) I misused its imports